components/tooltip/tooltip.js

```diff
@@ -650,6 +657,27 @@ class Tooltip extends RtlMixin(LitElement) {
 		return spaceAround;
 	}
 
+	async _determineIfTruncating() {
+		// if no resize has happened since truncation was previously calculated the result will not have changed
+		if (!this._resizeRunSinceTruncationCheck || !this.onlyShowIfTruncating) return;
+
+		const target = this._target;
+		const clone = target.cloneNode(true);
+		clone.removeAttribute('id');
+		clone.style.position = 'absolute';
+		if (this.getAttribute('dir') === 'rtl') {
+			clone.style.right = '-10000px';
```

Comment:
Although unlikely, content this size (at 19px font-size) would be possible to spill into the viewport.  I suspect this doesn't align with the driving use-case, but I have a sneaking feeling this is going to get used in all sorts of contexts, possibly even with user-authored text.
![image](https://user-images.githubusercontent.com/9042472/161622781-55655c09-009d-4a0b-8e01-828ee5c7a393.png)

The safest way to address this is probably to place the element in a containing element offscreen that doesn't show overflow or allow wrapping (like with `.d2l-offscreen`), and use `scrollWidth` instead of `offsetWidth`.

Another way might be to apply `visibility: hidden;` but I think it could result in horizontal scrollbar if the invisible text stretch beyond the opposite side of the viewport.